data/categories.txt

```diff
@@ -6420,7 +6428,7 @@ ru:^Автомойка
 ar:^مغسلة سيارات
 cs:^Myčka aut
 da:^Bilvask
-nl:^Car wash
+nl:^Car wash|auto was
```

Comment:
I think 'autowasstraat' or 'wasstraat' are more appropriate translations

Reply:
Yes auto was is probably more slang. I'll take your suggestion 👍